docs/sharing.md

```diff
@@ -561,4 +561,61 @@ Content-Type: application/json
 []
 ```
 
+### PUT /sharings/:sharing-id/io.cozy.files/:file-id/metadata
+
+This is an internal endpoint used by a stack to send the new metadata about a
+file that has changed.
+
+#### Request
+
+```http
+PUT /sharings/ce8835a061d0ef68947afe69a0046722/io.cozy.files/0c1116b028c6ae6f5cdafb949c088265/metadata HTTP/1.1
+Host: bob.example.net
+Accept: application/json
+Content-Type: application/json
+Authorization: Bearer ...
+```
+
+#### Response
+
+If only the metadata has changed (not the content), the response will be a
```

Comment:
If only -> Only if :)

Reply:
What I wanted to say is that if the metadata has changed, the response will be a 204, but if the metadata and the content (or just the content) have changed, the response will be a 200. In short: if "only the metadata has changed", then 204.